waku/common/wakubridge.nim

```diff
@@ -290,6 +291,6 @@ when isMainModule:
         address = conf.metricsServerAddress
         port = conf.metricsServerPort + conf.portsShift
       info "Starting metrics HTTP server", address, port
-      metrics.startHttpServer($address, Port(port))
+      startMetricsHttpServer($address, Port(port))
```

Comment:
You can remove the `when defined(insecure):` compile time option everywhere now that we can use the chronos http server in metrics.

Reply:
Right! Thanks.